oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -36,6 +37,15 @@ pub use channel::{ChannelEither, ChannelReader, ChannelWriter, ReadStatus};
 
 type Channels = Vec<Weak<channel::Channel>>;
 
+#[derive(Debug)]
+struct Node {
+    reference: NodeRef,
+    join_handle: Option<JoinHandle>,
```

Comment:
I guess this may become `HashSet<JoinHandle>` in the future, if / when we support multiple threads per node?

Reply:
Yeah, this can be expanded